config/http_config.go

```diff
@@ -177,6 +177,8 @@ type HTTPClientConfig struct {
 	// The omitempty flag is not set, because it would be hidden from the
 	// marshalled configuration when set to false.
 	FollowRedirects bool `yaml:"follow_redirects" json:"follow_redirects"`
+	// Extra Headers to be added on the Http Request
+	ExtraHeaders map[string]string `yaml:"extra_headers,omitempty" json:"extra_headers,omitempty"`
```

Comment:
Just to make a note of it here, over Slack, I suggested for this to be changed to just `headers`.

Reply:
Make sense.. changed... 
I still need to add the blocked headers...